js/src/settings/reconnectionLock.scss

```diff
@@ -0,0 +1,24 @@
+// The workaround for making the GLA menu links of WC Navigation appear as disabled
```

Comment:
> or setup some kind of disable class on the items?

The menu is rendered by React as well. Adding CSS class or setting `disabled` attribute could still be cleared when re-rendering.

> Regarding last-child. Would not be more accurate to use the href

I considered it, but since the menu is controlled by WC core, there are chances that it will be unreliable either way. Such as, some query strings in `href` are escaped, and their orders are not guaranteed. But on second thought, it's still a little better maintenance than the last-child method. I adjusted it in https://github.com/woocommerce/google-listings-and-ads/pull/1418/commits/aa8260028e76ca97611c4bd44e18ecec760e29c2.

Reply:
> I was thinking more about `&:not([href*="path=%2Fgoogle%2Fsettings"])` WDTY?

From the perspective of reliability, when using the `href` attribute to select the menu links but if the CSS selector fails to match the original targets due to an update in the future:
- Include-list (the current selector) has a higher chance of mismatch, so the targeting GLA menu links could be displayed as enabled.
- Exclude-list (the `&:not([href*="path=%2Fgoogle%2Fsettings"])` selector) has a higher chance of matching the menu link of GLA Settings and leads to the Settings link being displayed as disabled.

After comparing this, I chose to use the include-list approach to reduce the possibility of the GLA Settings link being displayed as disabled.